base/math.jl

```diff
@@ -1555,7 +1562,7 @@ sincos(a::Float16) = Float16.(sincos(Float32(a)))
 for f in (:sin, :cos, :tan, :asin, :atan, :acos,
           :sinh, :cosh, :tanh, :asinh, :acosh, :atanh,
           :exp, :exp2, :exp10, :expm1, :log, :log2, :log10, :log1p,
-          :exponent, :sqrt, :cbrt)
+          :exponent, :sqrt, :cbrt, :fourthroot)
```

Comment:
I think this might be the problem.

Reply:
I think you're right, this seems to solve it.

Thanks for the help!